src/ipfs/getPhotoGallery.ts

```diff
@@ -1,8 +1,8 @@
 import axios from 'axios';
 
-export const ipfsProvider = process.env.REACT_APP_INTERPLANETARY_FISSION_URL || 'https://hostless.dev';
-const username = process.env.REACT_APP_INTERPLANETARY_FISSION_USERNAME;
-const password = process.env.REACT_APP_INTERPLANETARY_FISSION_PASSWORD;
+export const ipfsProvider = process.env.INTERPLANETARY_FISSION_URL || 'https://runfission.com';
```

Comment:
the `REACT_APP_` here is actually intentional. Otherwise react can't find the env variables: https://github.com/fission-suite/ipfs-photo-gallery/issues/12#issuecomment-564142309

Reply:
Ok. But in Heroku, from our add-on, it is without REACT_APP. Does it just do this automatically?